contrib/win32/win32compat/termio.c

```diff
@@ -260,6 +260,18 @@ int
 syncio_close(struct w32_io* pio)
 {
 	debug4("syncio_close - pio:%p", pio);
+
+	/* Flush descriptor.*/
+	if (pio->write_details.pending) {
+		WaitForSingleObject(pio->write_overlapped.hEvent, INFINITE);
+
+		/* drain queued APCs */
+		SleepEx(0, TRUE);
```

Comment:
Is there a reason why you added this call to SleepEx to this is statement block?